src/index.jsx

```diff
@@ -23,6 +25,21 @@ root.render(
 	<StrictMode>
 		<DarkModeProvider>
 			<AppWithTheme />
+			<ToastContainer
+				limit={2}
+				position="top-right"
+				icon={true}
+				autoClose={3000}
```

Comment:
I would increase this to 4000, I had a hard time reading some of the longer messages in the time alloted.

Reply:
We will apply this change! Good thought!